services/bundlejs/bundlejs-package.service.js

```diff
@@ -0,0 +1,130 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  size: Joi.object({
+    compressedSize: Joi.string(),
+  }).required(),
+}).required()
+
+const keywords = ['node', 'bundlejs']
+
+const esbuild =
+  '<a href="https://github.com/evanw/esbuild" target="_blank" rel="noopener">esbuild</a>'
+const denoflate =
+  '<a href="https://github.com/hazae41/denoflate" target="_blank" rel="noopener">denoflate</a>'
+const bundlejs =
+  '<a href="https://bundlejs.com/" target="_blank" rel="noopener">bundlejs</a>'
+
+const documentation = `
+<p>
+ View ${esbuild} minified and ${denoflate} gzipped size of a package or selected exports, via ${bundlejs}.
+</p>
+`
+
+export default class BundlejsPackage extends BaseJsonService {
+  static category = 'size'
+
+  static route = {
+    base: 'bundlejs/package',
+    pattern: ':scope(@[^/]+)?/:packageName/:exports?',
+  }
```

Comment:
Based on our URL conventions https://github.com/badges/shields/blob/master/doc/badge-urls.md , I'd say:

- the base should be `bundlejs/size` - the thing we're actually displaying on the badge is a size, not a package
- `exports` should be a query param, not part of the route. It is optional and I'd expect it to be relatively infrequently used